containers/systemd-service

```diff
@@ -59,13 +57,22 @@ function run {
   fi
 }
 
+function sdproxy() {
+  SD_NOTIFY_PROXY=/usr/local/pf/bin/sdnotify-proxy
+  SD_SOCK=/tmp/$1.sock
+  CR_SOCK=/var/run/$1.sock
```

Comment:
IMO we use the same socket path in the container and outside to help things stay consistent

Reply:
I had an issue with that for the haproxy-portal. I had to move it in /var/run/ in the container to make it work.
It is when it has been introduce. I have to retest that part to be sure.

Reply:
I still do not know why but /tmp/xx.sock is failing with haproxy-portal but i was able to make it work with /usr/local/pf/var/run/ for exemple.

Reply:
In this case we can use the same value for both and use `/usr/local/pf/var/run/$1-systemd-notify.sock`

What do you think?

Reply:
Agree. I will try with all containers.